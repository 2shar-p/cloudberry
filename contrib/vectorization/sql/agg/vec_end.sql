DROP TABLE IF EXISTS trow;
DROP TABLE IF EXISTS tagg;
DROP TABLE IF EXISTS tnulls;
DROP TABLE IF EXISTS tempty;
