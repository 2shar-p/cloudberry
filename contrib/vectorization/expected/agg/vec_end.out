DROP TABLE IF EXISTS trow;
DROP
DROP TABLE IF EXISTS tagg;
DROP
DROP TABLE IF EXISTS tnulls;
DROP
DROP TABLE IF EXISTS tempty;
DROP
